R/query_dataset.R

```diff
@@ -0,0 +1,37 @@
+#' Query a data set using GET and a query URL
+#'
+#' @description
+#' This function provides a method for generating and sending a URL based data query to the
+#' EES API. As a minimum, it requires the dataset_id and indicators flags to be provided.
+#'
+#' @param dataset_id ID of data set to be connected to
```

Comment:
Worth looking at @inheritParams that you can do in roxygen, to see if we can have an automatic link between the params that are shared in this and the `api_url_query()` function?